atomic_reactor/plugins/exit_sendmail.py

```diff
@@ -123,13 +124,19 @@ def __init__(self, tasker, workflow,
         else:
             self.log.info("Koji task ID: %s", self.koji_task_id)
 
-        try:
-            self.koji_build_id = self.workflow.exit_results.get(KojiPromotePlugin.key)
-        except Exception:
+        valid_id = False
+        for key in [KojiImportPlugin.key, KojiPromotePlugin.key]:
```

Comment:
original code was written that way. sendmail used a try/catch pair while koji_tag just did an assignment.